### PR TITLE
quickSort c++ code added

### PR DESCRIPTION
**Pull request by kumaryogesh17**
C++ code of QuickSort

### CODE REVIEW

File: algorithms/QuickSort/quickSort.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#include<bits/stdc++.h>
```
Comment (ademclk) [inline algorithms/QuickSort/quickSort.cpp:1]: Please use standard library for C++ or rewrite the code in C.

**Review by ademclk — CHANGES_REQUESTED** [review]
Change source file or rewrite in C.

**Comment by kumaryogesh17** [comment]
updated standard library and code

**Review by ademclk — APPROVED** [review]
I will change it little bit.

**Pull request merged**